src/Controllers/ImportController.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace NerdsAndCompany\Schematic\Controllers;
+
+use Craft;
+use NerdsAndCompany\Schematic\Models\Data;
+use NerdsAndCompany\Schematic\Schematic;
+use craft\errors\WrongEditionException;
+
+/**
+ * Schematic Import Command.
+ *
+ * Sync Craft Setups.
+ *
+ * @author    Nerds & Company
+ * @copyright Copyright (c) 2015-2018, Nerds & Company
+ * @license   MIT
+ *
+ * @see      http://www.nerds.company
+ */
+class ImportController extends Base
+{
+    public $force = false;
+
+    /**
+     * {@inheritdoc}
+     *
+     * @return array
+     */
+    public function options($actionID): array
+    {
+        return array_merge(parent::options($actionID), ['force']);
+    }
+
+    /**
+     * Imports the Craft datamodel.
+     *
+     * @return int
+     */
+    public function actionIndex(): int
+    {
+        if (!file_exists($this->file)) {
+            Schematic::error('File not found: '.$this->file);
+
+            return 1;
+        }
+
+        $dataTypes = $this->getDataTypes();
+        $this->importFromYaml($dataTypes);
+        Schematic::info('Loaded schema from '.$this->file);
+
+        return 0;
+    }
+
+    /**
+     * Import from Yaml file.
+     *
+     * @param string $dataTypes The data types to import
+     *
+     * @throws Exception
+     */
+    private function importFromYaml($dataTypes): void
+    {
+        $this->disableLogging();
+        $yaml = file_get_contents($this->file);
```

Comment:
Doesn't Craft have a file helper for this anymore?

Reply:
Doesn't seem like it:
https://github.com/craftcms/cms/blob/develop/src/helpers/FileHelper.php